controllers/spec/common.go

```diff
@@ -1124,9 +1122,12 @@ func getProcessJavaRuntimeArgs(name, packageName, clusterName, logLevel, details
 		"-Dpulsar.function.log.dir=logs/functions",
 		"-Dpulsar.function.log.file=" + fmt.Sprintf("%s-${%s}", name, EnvShardID),
 		setLogLevel,
-		"-Xmx" + getDecimalSIMemory(memory),
-		"-Xms" + getDecimalSIMemory(xmsMemory),
+		"-XX:InitialRAMPercentage=20",
+		"-XX:MaxRAMPercentage=40",
```

Comment:
does this `MaxRAMPercentage` means `-Xmx`?

Reply:
Yes.